server/controllers/plaid.js

```diff
@@ -1,18 +1,54 @@
 const axios = require('axios');
 
 const Category = require('../../database/models/category');
+const User = require('../../database/models/user');
 const transactionsController = require('./transactions');
 const accountsController = require('./accounts');
 const helpers = require('../helpers');
 
 require('dotenv').config({ path: `${__dirname}/../../.env` });
 
+const { PLAID_CLIENT_ID, PLAID_SECRET } = process.env;
+
 module.exports = {
   connect: {
```

Comment:
Good job on this! Once we hit MVP, I would recommend refactoring. This method does a LOT, and we will want to have this broken into a few different components in case we need to reuse parts of it, or simply want make things more legible.